src/lib/organisms/methods.svelte

```diff
@@ -179,12 +179,19 @@
     outline: none;
   }
 
+  article a {
+    display: block;
+  }
+
   .grid h2 {
     font-family: var(--vtPrimaryFont);
     color: var(--vtDarkBlue);
     font-size: 19px;
```

Comment:
Changes static font-size to dynamic font-size

Reply:
Done